tests/functional/csp-cdn.ts

```diff
@@ -0,0 +1,13 @@
+require.config({
+	packages: [
+		{name: 'amdApp', location: './amdApp'},
+		{name: 'dojo', location: '//ajax.googleapis.com/ajax/libs/dojo/1.10.4/dojo'}
```

Comment:
I can certainly do that, but I wasn't sure if these tests where going to stay or not, seeing as how they don't pass.

I think I would have to disable instrumentation to get the CSP tests to run.